ui/apps/platform/src/Containers/NetworkGraph/common/NetworkPolicies.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Button, SelectOption, Stack, StackItem } from '@patternfly/react-core';
+import { CodeEditor, Language } from '@patternfly/react-code-editor';
+
+import download from 'utils/download';
+import SelectSingle from 'Components/SelectSingle';
+
+type NetworkPolicy = {
+    name: string;
+    yaml: string;
+};
+
+type NetworkPoliciesProps = {
+    networkPolicies: NetworkPolicy[];
+};
+
+const downloadYAMLHandler = (fileContent: string) => () => {
+    download('network-policy.yml', fileContent, 'yml');
+};
+
+function NetworkPolicies({ networkPolicies }: NetworkPoliciesProps): React.ReactElement {
+    const [selectedNetworkPolicy, setSelectedNetworkPolicy] = React.useState<
+        NetworkPolicy | undefined
+    >(networkPolicies[0]);
+
+    function handleSelectedNetworkPolicy(_, value: string) {
+        const newlySelectedNetworkPolicy = networkPolicies.find(
+            (networkPolicy) => networkPolicy.name === value
+        );
+        setSelectedNetworkPolicy(newlySelectedNetworkPolicy);
+    }
+
+    return (
+        <Stack hasGutter>
+            <StackItem>
+                <SelectSingle
+                    id="search-filter-attributes-select"
+                    value={selectedNetworkPolicy?.name || ''}
+                    handleSelect={handleSelectedNetworkPolicy}
+                    placeholderText="Select a network policy"
+                >
+                    {networkPolicies.map((networkPolicy) => {
+                        return (
+                            <SelectOption key={networkPolicy.name} value={networkPolicy.name}>
+                                {networkPolicy.name}
+                            </SelectOption>
+                        );
+                    })}
+                </SelectSingle>
+            </StackItem>
+            {selectedNetworkPolicy && (
+                <StackItem>
+                    <CodeEditor
+                        isDarkTheme
```

Comment:
Make this configurable.

```suggestion
                        isDarkTheme={showDarkTheme}
```